From: David Bate <david@bate.org.uk>
Date: Sat, 11 Jan 2014 11:08:04 -0600
Subject: Disable gpu skinning for skeletal models

This feature is buggy and sometimes causes models to completely
lack animation and can cause very high frame rates.  Xonotic
developers are aware of this and disabling this feature is the
temporary work around. See http://dev.xonotic.org/issues/1459
---
 gl_rmain.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gl_rmain.c b/gl_rmain.c
index 38f2eaa..7b326b9 100644
--- a/gl_rmain.c
+++ b/gl_rmain.c
@@ -183,7 +183,7 @@ cvar_t r_viewscale_fpsscaling_stepsize = {CVAR_SAVE, "r_viewscale_fpsscaling_ste
 cvar_t r_viewscale_fpsscaling_stepmax = {CVAR_SAVE, "r_viewscale_fpsscaling_stepmax", "1.00", "largest adjustment to hit the target framerate (this value prevents wild overshooting of the estimate)"};
 cvar_t r_viewscale_fpsscaling_target = {CVAR_SAVE, "r_viewscale_fpsscaling_target", "70", "desired framerate"};
 
-cvar_t r_glsl_skeletal = {CVAR_SAVE, "r_glsl_skeletal", "1", "render skeletal models faster using a gpu-skinning technique"};
+cvar_t r_glsl_skeletal = {CVAR_SAVE, "r_glsl_skeletal", "0", "render skeletal models faster using a gpu-skinning technique"};
 cvar_t r_glsl_deluxemapping = {CVAR_SAVE, "r_glsl_deluxemapping", "1", "use per pixel lighting on deluxemap-compiled q3bsp maps (or a value of 2 forces deluxemap shading even without deluxemaps)"};
 cvar_t r_glsl_offsetmapping = {CVAR_SAVE, "r_glsl_offsetmapping", "0", "offset mapping effect (also known as parallax mapping or virtual displacement mapping)"};
 cvar_t r_glsl_offsetmapping_steps = {CVAR_SAVE, "r_glsl_offsetmapping_steps", "2", "offset mapping steps (note: too high values may be not supported by your GPU)"};
